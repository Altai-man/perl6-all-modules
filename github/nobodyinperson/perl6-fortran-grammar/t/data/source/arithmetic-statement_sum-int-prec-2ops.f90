1_wp + 1_wp
