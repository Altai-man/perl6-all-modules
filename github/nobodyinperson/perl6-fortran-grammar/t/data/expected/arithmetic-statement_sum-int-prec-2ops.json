{
  "arithmetic-operator": [
    "+"
  ],
  "value-returning-code": [
    {
      "in-place": {
        "atomic": {
          "number": {
            "integer": {
              "digits": "1",
              "precision-spec": {
                "name": "wp"
              }
            }
          }
        }
      }
    },
    {
      "in-place": {
        "atomic": {
          "number": {
            "integer": {
              "digits": "1",
              "precision-spec": {
                "name": "wp"
              }
            }
          }
        }
      }
    }
  ]
}
